qiling/debugger/gdb/gdb.py

```diff
@@ -65,6 +65,11 @@ def __init__(self, ql, ip, port):
         else:
             self.entry_point = self.ql.os.entry_point
 
+        # Only part of the binary file is debugged.
+        if self.ql.entry_point is not None and self.ql.exit_point is not None:
+            self.entry_point=self.ql.entry_point
```

Comment:
please put a space before and after `=`  (so " = ", not "=").

Reply:
Oh, my bad. Fixed now.